### PR TITLE
Regression: Fix marketplace releases tab crash bug

### PR DESCRIPTION
**Pull request by rique223**
## Proposed changes (including videos or screenshots)

Fixed a bug where RC would crash because the marketplace releases tab was trying to display undefined data from manually installed apps. 
Demo gif:
app-releases-tab-crash-error

### CODE REVIEW

**Comment by rique223** [comment]
> If they only provide one link, such as terms but not the privacy, I would vote we show whichever one we get back. Thoughts?

Makes sense, when I did this logic I believed both informations were mandatory. Will adapt it in a sec.

**Review by graywolf336 — DISMISSED** [review]

**Review by geekgonecrazy — APPROVED** [review]

File: apps/meteor/client/views/admin/apps/AppSecurity.tsx [hunk 1]
```diff
@@ -75,12 +75,16 @@ const AppSecurity: FC<AppSecurityProps> = ({ privacyPolicySummary, appPermission
...
 						<Box display='flex' flexDirection='column'>
-							<Box is='a' href={tosLink}>
-								Terms of use
-							</Box>
-							<Box is='a' href={privacyLink}>
-								Privacy policy
-							</Box>
+							{tosLink && (
+								<Box is='a' href={tosLink}>
+									Terms of use
```
Comment (ggazzo) [inline apps/meteor/client/views/admin/apps/AppSecurity.tsx:80]: could you replace this hard text ? use translations keys instead

**Review by ggazzo — CHANGES_REQUESTED** [review]

File: apps/meteor/client/views/admin/apps/AppSecurity.tsx (hunk 1 above)
Comment (rique223) [reply apps/meteor/client/views/admin/apps/AppSecurity.tsx:80]: Solved

Comment (ggazzo) [reply apps/meteor/client/views/admin/apps/AppSecurity.tsx:80]: solved where my friend?

**Pull request merged**